src/engine/sparqlExpressions/SparqlExpressionPimpl.h

```diff
@@ -111,6 +118,9 @@ class SparqlExpressionPimpl {
       uint64_t inputSizeEstimate,
       const std::optional<Variable>& primarySortKeyVariable);
 
+  std::vector<PrefilterExprVariablePair> getPrefilterExpressionForMetadata(
+      bool isNegated = false) const;
```

Comment:
I think for the `Pimpl` , we don't need the `isNegated` argument, drop it.
(only used in the actual implementation.